### DISCUSSION TITLE
PhotoPrism not loading on LG TVs

### DISCUSSION THREAD

**Discussion by 4193883-eng** [photoprism/photoprism#3587] [Q&A]
#### 1. What is not working as documented?

PhotoPrism not loading on LG TVs

#### 2. How can we reproduce it?

1. Go to PhotoPrism
2. See a non-ending scrollbar

#### 3. What behavior do you expect?

It loading 🤷 

#### 4. What could be the cause of your problem?

Maybe something not implemented in LG's webbrowser

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

![IMG_4190](https://github.com/photoprism/photoprism/assets/83756034/7c825f47-a9c9-4135-9dc3-cd55277089f4)

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: AMD64, ARM64, ARMv7,...: PhotoPrism's demo website and my own AMD64

with this compose: https://paste.ee/p/YafKX

(b) Database Type & Version: MariaDB, MySQL, SQLite,...: listed in the compose file

(c) Operating System Types & Versions: Linux, Windows, Android,...: WebOS

(d) Browser Types & Versions: Browser on a LG TV

(e) Ad Blockers, Browser Plugins, and/or Firewall Software? No, because it is a TV

#### 7. On what kind of device is PhotoPrism installed?

This is especially important if you are reporting a performance, import, or indexing issue. You can skip this if you're reporting a problem you found in our public demo, or if it's a completely unrelated issue, such as incorrect page layout.

(a) Device / Processor Type: Raspberry Pi 4, Intel Core i7-3770, AMD Ryzen 7 3800X,...
Your demo website, and my Intel Core i7-10700K
(b) Physical Memory & Swap Space in GB
32 GB and 32 GB in swap
(c) Storage Type: HDD, SSD, RAID, USB, Network Storage,...
Compose file on an NVME SSD, and data store on a RAID 5 of 4 disks
(d) Anything else that might be helpful to know?

There is no devtools on LG TVs

#### 8. Do you use a Reverse Proxy, Firewall, VPN, or CDN?

I use Cloudflare Tunnels and whatever your demo uses

**Comment by lastzero** [comment]
The LG browser might NOT fully support JS or could be otherwise incompatible. Maybe you can install an alternative browser on your device.
